tox.ini

```diff
@@ -1,8 +1,7 @@
 [tox]
 envlist =
-       py{27,34,35,36,37}-django111
-       py{34,35,36,37}-django20
-       py{35,36,37}-django21
+       lint
```

Comment:
We can run the linting once in a separate `tox` environment instead of rerunning it for every environment. This cuts down on build time. (azavea/django-amazon-ses#61)